modules/import/csv_file.py

```diff
@@ -140,7 +140,7 @@ def __parse_file(self):
             for row in csvreader:
                 # append all lines as-is case-wise
                 # unicode(str, errors='ignore') causes all invalid characters to be stripped out
-                values.append([unicode(value.strip(), errors='ignore') for value in row])
+                values.append([str(value.strip(), errors='ignore') for value in row])
```

Comment:
This is something I will look at in mass. I have an encoding conversion helper method in the core that will convert unknown data types to unicode strings safely for printing, storing, etc.